### ISSUE TITLE
Oauth endpoint

### ISSUE THREAD

**Issue by danielsz** [Weebly/node-webhook-client#4]
Compared with the php example application, the Oauth endpoints do not match. 

https://github.com/Weebly/node-webhook-client/blob/master/app/oauth-router.js#L45
```sh
https://api.weebly.com/app-center/authorize
```
vs.

https://github.com/Weebly/weebly-client-php/blob/0.1/src/Weebly/WeeblyClient.php#L119
```sh
https://www.weebly.com/app-center/oauth/authorize
```

**Comment by puddi** [comment]
@danielsz: looks like that get request is unnecessary. we'll get rid of it. thanks for pointing it out!

**Comment by danielsz** [comment]
Only unnecessary if you don't plan to support unauthenticated users coming from the app (as opposed to users from the app center who are already authenticated).

**Comment by puddi** [comment]
How would the call removed in #5 handle that case? It didn't GET with anything, so it was effectively a check to see "does the server exist".

**Comment by danielsz** [comment]
True. I didn't mean that this line alone was handling the case of unauthenticated users, just that it would be handy to add support for that use case. Sorry for the misunderstanding.
